community/kernel/src/test/java/org/neo4j/kernel/impl/storemigration/MigrationTestUtils.java

```diff
@@ -148,17 +148,24 @@ public static void prepareSampleLegacyDatabase( String version, FileSystemAbstra
 
     public static File findFormatStoreDirectoryForVersion( String version, File targetDir ) throws IOException
     {
-        switch ( version )
+        if ( version.equals( StandardV2_3.STORE_VERSION ) )
```

Comment:
Feels a bit harsh to introduce a public method in `StoreVersion` just for this. We can make these methods in `MigrationTestUtils` accept `StoreVersion` instead of `String`. It'll give more type safety and possibility to use `switch`. Do you mind if I do this in 3.1 as a separate change?